include/wasm/WasmModule.h

```diff
@@ -121,8 +142,8 @@ class WasmModule
     int stdoutMemFd;
     ssize_t stdoutSize;
 
-    std::mutex moduleMemoryMutex;
```

Comment:
Now that we use a thread-safe dlmalloc we don't need to do our own locking on the module memory.